docs/api-v0.5/TableAPI.md

```diff
@@ -315,3 +315,10 @@ Whether a column is currently being resized.
 
 type: `bool`
 
+
+### `bufferRowCount`
+
+The number of rows outside the viewport to prerender. Defaults to roughly half
+of the number of visible rows.
+
+type: `number`
```

Comment:
We aren't supporting this for v0.5, thats been deprecated. 
Can you remove this change

@wcjordan we should remove all the 0.5 documentation..